Formula/u/uffizzi.rb

```diff
@@ -0,0 +1,23 @@
+class Uffizzi < Formula
+  desc "Self-serve developer platforms in minutes, not months with k8s virtual clusters"
+  homepage "https://uffizzi.com"
+  url "https://github.com/UffizziCloud/uffizzi_cli/archive/refs/tags/v2.0.35.tar.gz"
+  sha256 "95a541b167307aac7a69416dd621134eefae4f910722fb14d87da0b0b5b116bf"
+  license "Apache-2.0"
+
+  uses_from_macos "ruby"
+
+  def install
+    system "gem", "install", "uffizzi-cli", "-v", "2.0.35", "--no-document", "--install-dir", libexec
```

Comment:
This has undetermined dependencies. Please install from the source archive you're downloading instead of just `gem install`

Reply:
Sure! Thanks for help!

Reply:
@SMillerDev I've updated the formula to install dependencies from the sources, but the installation fails again on MacOs 11. It's not clear from the pipeline [logs](https://github.com/Homebrew/homebrew-core/actions/runs/6144833025/job/16671135398) what exactly is wrong.